packages/fuels-rs/src/lib.rs

```diff
@@ -28,6 +28,10 @@ pub mod signers {
     pub use fuels_signers::*;
 }
 
+pub mod test_helpers {
```

Comment:
I didn't think about this before, this will be breaking change unless `DEFAULT_INITIAL_BALANCE` is also expotred from `mod core`